src/containers/Home/HomeHero/HomeHero.scss

```diff
@@ -127,6 +127,7 @@ $title-width: 592px;
     height: 64px;
     justify-content: center;
     width: 64px;
+    transition: background-color 0s, color 0s;
```

Comment:
hi can you explain why this addition will solve the issue? I don't see any other places that would affect the transition here

Reply:
While inspecting the website, The default transition was 0.3s and so if we change it to 0s the lag gets fixed.

![Home _ thenewboston - Google Chrome 14-06-2021 21_33_13 (2)_LI](https://user-images.githubusercontent.com/85382825/121923609-bc9aa680-cd58-11eb-9c7f-ac621f9474e4.jpg)